gradle/libs.versions.toml

```diff
@@ -0,0 +1,45 @@
+[metadata]
+format.version = "1.1"
+
+[versions]
+assertj = "3.23.1"
+edc = "0.0.1-SNAPSHOT"
+jackson = "2.14.2"
+jetbrainsAnnotation = "24.0.1"
+jupiter = "5.9.2"
+mockito = "5.2.0"
+
+[libraries]
+edc-api-management = { module = "org.eclipse.edc:management-api", version.ref = "edc" }
+edc-api-management-config = { module = "org.eclipse.edc:management-api-configuration", version.ref = "edc" }
+edc-api-observability = { module = "org.eclipse.edc:api-observability", version.ref = "edc" }
+edc-boot = { module = "org.eclipse.edc:boot", version.ref = "edc" }
+edc-config-filesystem = { module = "org.eclipse.edc:configuration-filesystem", version.ref = "edc" }
+edc-core-connector = { module = "org.eclipse.edc:connector-core", version.ref = "edc" }
+edc-core-controlplane = { module = "org.eclipse.edc:control-plane-core", version.ref = "edc" }
+edc-core-jersey = { module = "org.eclipse.edc:jersey-core", version.ref = "edc" }
+edc-core-jetty = { module = "org.eclipse.edc:jetty-core", version.ref = "edc" }
+edc-dpf-framework = { module = "org.eclipse.edc:data-plane-framework", version.ref = "edc" }
+edc-dpf-selector-client = { module = "org.eclipse.edc:data-plane-selector-client", version.ref = "edc" }
+edc-dpf-selector-core = { module = "org.eclipse.edc:data-plane-selector-core", version.ref = "edc" }
+edc-dpf-selector-spi = { module = "org.eclipse.edc:data-plane-selector-spi", version.ref = "edc" }
+edc-dpf-transferclient = { module = "org.eclipse.edc:data-plane-transfer-client", version.ref = "edc" }
+edc-ext-azure-cosmos-core = { module = "org.eclipse.edc:azure-cosmos-core", version.ref = "edc" }
+edc-ext-azure-test = { module = "org.eclipse.edc:azure-test", version.ref = "edc" }
+edc-ext-http = { module = "org.eclipse.edc:http", version.ref = "edc" }
+edc-iam-mock = { module = "org.eclipse.edc:iam-mock", version.ref = "edc" }
+edc-ids = { module = "org.eclipse.edc:ids", version.ref = "edc" }
+edc-junit = { module = "org.eclipse.edc:junit", version.ref = "edc" }
+edc-spi-catalog = { module = "org.eclipse.edc:catalog-spi", version.ref = "edc" }
+edc-spi-core = { module = "org.eclipse.edc:core-spi", version.ref = "edc" }
+edc-spi-ids = { module = "org.eclipse.edc:ids-spi", version.ref = "edc" }
+edc-spi-web = { module = "org.eclipse.edc:web-spi", version.ref = "edc" }
+edc-util = { module = "org.eclipse.edc:util", version.ref = "edc" }
+edc-versions = { module = "org.eclipse.edc:edc-versions", version.ref = "edc" }
+
+[bundles]
+edc-connector = [ "edc-boot", "edc-core-connector", "edc-core-jersey", "edc-core-controlplane", "edc-api-observability" ]
+edc-dpf = [ "edc-dpf-transferclient", "edc-dpf-selector-client", "edc-dpf-selector-spi", "edc-dpf-selector-core", "edc-dpf-framework" ]
+
+[plugins]
+shadow = { id = "com.github.johnrengelman.shadow", version = "8.0.0" }
```

Comment:
maybe this should (eventuall) go to the `root` version catalog?

Reply:
maybe in the future, but currently gradle version is not the same for all the repos so it could cause trouble.